### PR TITLE
Add serverless eval

### PR DESCRIPTION
**Pull request by sarupbanskota**
The blog post isn't live yet, but should be live soon!

### CODE REVIEW

File: serverless-eval/now.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "version": 2,
+  "env": {
+    "SLACK_CLIENT_ID": "@slack-client-id",
+    "SLACK_CLIENT_SECRET": "@slack-client-secret"
+  },
+  "builds": [{
+    "src": "index.js",
+    "use": "@now/node",
+  }, {
+    "src": "validate.js",
```
Comment (TejasQ) [inline serverless-eval/now.json]: Use a wildcard for the same builder?

File: serverless-eval/oauth.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+const fetch = require("node-fetch");
+const querystring = require("querystring");
+const { parse } = querystring;
```
Comment (TejasQ) [inline serverless-eval/oauth.js]: You can just destructure on line 2.

File: serverless-eval/oauth.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  const client_secret = process.env.SLACK_CLIENT_SECRET;
+  const authHeader = 'Basic ' + Buffer.from(`${client_id}:${client_secret}`).toString('base64');
+
+  // Hit oauth.access for access_token
+  const oauthAccess = await fetch('https://slack.com/api/oauth.access', {
+    method: 'POST',
+    body: querystring.stringify({ code }),
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+      'Authorization': authHeader
```
Comment (TejasQ) [inline serverless-eval/oauth.js]: You can use concise properties syntax if you rename `authHeader` to `Authorization`.

**Comment by TejasQ** [comment]
@sarupbanskota don't forget to rename the folder from `serverless-` to `slack-` 😄

**Comment by sarupbanskota** [comment]
@TejasQ there's something broken now with the following section:

```js  
const urlQueryString = req.url.replace(/^.*?/, '');
const code = parse(urlQueryString).code;
```

I'm investigating; should be good to go after!

**Comment by TejasQ** [comment]
Thanks, @sarupbanskota!

**Pull request merged**